cli/azd/internal/global_command_options.go

```diff
@@ -20,4 +20,9 @@ type GlobalCommandOptions struct {
 	// AZURE_DEV_COLLECT_TELEMETRY is set to 'no'.
 	// Defaults to true.
 	EnableTelemetry bool
+
+	// Generates platform-agnostic help for use on static documentation sites
```

Comment:
Do you think `StaticHelp` is a better naming? Or does `PlatformAgnosticHelp` more closely represent what we want here?